docs/spec/errors/error.json

```diff
@@ -55,26 +57,27 @@
         },
         "log": {
             "type": ["object", "null"],
+            "description": "Additional information added when logging the error.",
             "properties": {
                 "level": {
-                    "description": "The record severity.",
+                    "description": "The severity of the record.",
                     "type": ["string", "null"],
                     "default": "error",
                     "enum": ["debug", "info", "warning", "error", "fatal", null],
                     "maxLength": 1024
                 },
                 "logger_name": {
-                    "description": "The name of the logger which created the record.",
+                    "description": "The name of the used logger instance.",
                     "type": ["string", "null"],
                     "default": "default",
                     "maxLength": 1024
                 },
                 "message": {
-                    "description": "The exception's error message.",
+                    "description": "The additionally logged error message.",
                     "type": "string"
                 },
                 "param_message": {
-                    "description": "A parametrized message. E.g. Could not connect to %s. The property message is still required, and should be equal to the param_message, but with placeholders replaced. In some situations the param_message is used to group errors together. The string is not interpreted, so feel free to use whichever placeholders makes sense in the client languange.",
+                    "description": "Equal to message, but with placeholders replaced.",
```

Comment:
The original description of `param_message` here is better than the one in the `fields.yml`. I suggest we update the `fields.yml` here instead.